source/Playnite/Scripting/PowerShell/PowerShell.cs

```diff
@@ -140,15 +140,12 @@ public object GetVariable(string name)
 
         public CommandInfo GetFunction(string name)
         {
-            var command = powershell
-                .AddCommand("Get-Command")
-                .AddArgument(name)
-                .AddParameter("PassThru")
-                .AddParameter("ErrorAction", ActionPreference.SilentlyContinue)
-                .Invoke<CommandInfo>().FirstOrDefault();
-            powershell.Streams.ClearStreams();
-            powershell.Commands.Clear();
-            return command;
+            if (module == null)
+            {
+                return null;
+            }
+            CommandInfo command;
+            return module.ExportedCommands.TryGetValue(name, out command) ? command : null;

```

Comment:
I decided to use [`ExportedCommands`](https://docs.microsoft.com/en-us/dotnet/api/system.management.automation.psmoduleinfo.exportedcommands?view=powershellsdk-7.0.0#System_Management_Automation_PSModuleInfo_ExportedCommands) instead of [`ExportedFunctions`](https://docs.microsoft.com/en-us/dotnet/api/system.management.automation.psmoduleinfo.exportedfunctions?view=powershellsdk-7.0.0#System_Management_Automation_PSModuleInfo_ExportedFunctions). `ExportedCommands` appears to be a superset of `ExportedFunctions`. I think this will capture someone defining something as a cmdlet too and not just functions. All the functions do correctly appear in this list also